package.json

```diff
@@ -1,32 +1,32 @@
 {
-  "name": "ember-plaid",
-  "version": "0.0.0",
-  "description": "The default blueprint for ember-cli addons.",
+  "name": "ember-plaid-v2",
+  "version": "0.1.0",
+  "description": "Plaid Link V2 for your Ember app",
   "keywords": [
     "ember-addon"
   ],
   "license": "MIT",
-  "author": "",
+  "author": "Aubrey Knudson, based on original code from Jason Kriss",
```

Comment:
nit: should be an array of email addresses or objects: https://docs.npmjs.com/files/package.json#people-fields-author-contributors